public/js/app.js

```
@@ -3562,4 +3498,12 @@ function filterClicksChart(days) {
     });
 
     renderClicksChart(processClicksOverTime(filtered));
-}
\ No newline at end of file
+}
+
+socket.on('analyticsUpdate', (newData) => {
```

ESLint

[error] 'socket' is not defined.

(no-undef)